src/components/label-pill/LabelPill.scss

```diff
@@ -0,0 +1,51 @@
+@import '../../styles/variables';
+
+.bdl-LabelPill {
+    padding: 3px;
+    color: $bdl-gray;
+    font-size: 10px;
+    line-height: 10px;
+    background-color: $bdl-gray-10;
+    border: 1px solid;
+    border-color: $bdl-gray-20;
+    border-radius: 10px;
+
+    &.bdl-LabelPill--warning {
+        background-color: $bdl-yellorange-10;
+        border-color: $bdl-yellorange;
+    }
+
+    &.bdl-LabelPill--error {
+        background-color: $bdl-watermelon-red-10;
+        border-color: $bdl-watermelon-red;
+    }
+
+    &.bdl-LabelPill--success {
+        background-color: $bdl-green-light-10;
+        border-color: $bdl-green-light;
+    }
+
+    &.bdl-LabelPill--alert {
+        background-color: $bdl-watermelon-red-10;
+        border-color: $bdl-watermelon-red;
+    }
+
+    &.bdl-LabelPill--highlight {
+        background-color: $bdl-light-blue-10;
+        border-color: $bdl-light-blue;
+    }
+
+    &.bdl-LabelPill--info {
+        background-color: $bdl-light-blue-10;
+        border-color: $bdl-light-blue;
+    }
+}
+
+.bdl-LabelPill-iconContent {
```

Comment:
Do we need these classes? Can they just be importable mixins? Or let the parent app deal with it.

Reply:
these classes define the content types and spacing specific to this component. I don't think any other component defines these specific values, so they aren't generic. 

another idea was to borrow an approach like we use in the media card component. if we created "sub-components" of LabelPill with these styles, the pattern could look like:

```
<LabelPill>
    <LabelPill.Text>{'some text content'}</LabelPill.Text>
</LabelPill>
```

having mixins seemed a bit tricky to me, because it would require the user to create some class for the content, then use the mixin in that class, which is an additional step. Here, we document the usage and class names in examples, so the user can follow this approach without adding any additional classes

Reply:
+2 for the sub-comp approach

Reply:
Throwing out another idea:

Since most of the time this is used with text content, having to add an inner component seems like a little extra work. Could we instead have default padding & alignment for text and then an `--round` modifier + prop to force it to be a circle for the icon use case?

Reply:
@alexkrolick this could work. One additional use case that was proposed in a design chat was to have an icon has text which is right next to the icon, i.e., no additional padding. that would be a bit in conflict with a default padding approach.

I'm inclined to try the sub-comp approach as it gives us clear error messaging on usage, and would likely allow for us to:

- more easily specify sizing for this pill to account for future sharing in other contexts, like having a prop for pill size,
- add a "mode" prop similar to icon bars in macOS where you can specify both the icon **and** text, but use a "text only" mode in some contexts
- extend the sub-component list for cases where we have text/icon combos that follow different rules

![Screen Shot 2020-02-06 at 2 26 44 PM](https://user-images.githubusercontent.com/3056447/73971357-ba1a7800-48ec-11ea-93aa-73ccea9c2ea6.png)

Reply:
The subcomponent approach sounds good for the icon + text mode, or even this avatar pill:

<img width="100" alt="Screen Shot 2020-02-06 at 14 29 34" src="https://user-images.githubusercontent.com/1571667/73984483-93a90c80-48ed-11ea-9be0-bec3a25fa117.png">